bower.json

```diff
@@ -1,25 +1,29 @@
 {
-  "name": "purescript-parsing-repetition",
```

Comment:
The churn in this file is due to `spago bump-version` replacing the `bower.json`.